### DISCUSSION TITLE
Latest version (3.4.1) not available in F-Droid

### DISCUSSION THREAD

**Discussion by rambalachandran** [KRTirtho/spotube#1190] [Q&A] [answered]
Hi,
I prefer to have android app updates from F-Droid. However I see that latest version `3.4.1` is not released there. Any plans to release the new version in F-Droid soon?

**Comment by KRTirtho** [comment, marked as answer]
Yeah it's available now

**Reply by rambalachandran** [reply]
@KRTirtho Thank you. It is a wonderful piece of software
